master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
```

Comment:
Are you suggesting I add that 2nd sentence you have there? or something mentioning that info?